index.html

```diff
@@ -5,11 +5,16 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <link rel="stylesheet" href="styles.css" />
-        <script src="game.js" defer></script>
+        <script src="game.js" defer type="module"></script>
```

Comment:
BTW ,i check it , and if you use type="module" then all the code in the file is in strict mode. so i mislead  you about this approach: it is just good to use type module , and you need to add this "use strict" in this case.